changelog/11775.improvement.md

```diff
@@ -0,0 +1 @@
+Docker container builds for arm64 (e.g. to allow running Rasa inside docker on M1/M2).
```

Comment:
Nit:
Should start with the verb which illustrates if we are adding new functionality, removing it or updating existing one.